src/firmware/app/control/trajectory_planner_test.cpp

```diff
@@ -1106,9 +1120,14 @@ TEST_F(TrajectoryPlannerTest,
         EXPECT_FLOAT_EQ(
             velocity_trajectory.y_velocity[i],
             position_trajectory.linear_speed[i] * static_cast<float>(directions[i].y()));
+        if (velocity_trajectory.angular_velocity[i] > max_angular_speed)
+        {
+            max_angular_speed = velocity_trajectory.angular_velocity[i];
+        }
         EXPECT_NEAR(velocity_trajectory.time_profile[i],
                     position_trajectory.time_profile[i], 0.00001);
     }
+    EXPECT_GT(max_angular_speed, 1.0f);
```

Comment:
Similar comment here about upper-bounding this number.

Reply:
fixed